packages/teams-js/src/public/pages.ts

```diff
@@ -1,10 +1,8 @@
 import {
-  Communication,
```

Comment:
All of the changes that were removed in this file were only added for Sharepoint scenarios which are no longer supported. As with the above example, we could choose to gate these behind an `allowMessageProxy` check if we wanted to be safer.